tools/js-sdk-release-tools/src/common/types.ts

```diff
@@ -1,4 +1,5 @@
 export enum SDKType {
+    None = 'None',
```

Comment:
is there a real case when we get a None type for SDK?

Reply:
removed